src/bindings/html/shims.js

```diff
@@ -27,3 +27,21 @@ export function getDirection(code) {
   return ['ar', 'he', 'fa', 'ps', 'ur'].indexOf(tag) >= 0 ?
     'rtl' : 'ltr';
 }
+
+// Opera and Safari don't support it yet
+if (navigator.languages === undefined) {
+  navigator.languages = [navigator.language];
+}
+
+// Safari 9 and iOS 9 does not support Intl
+if (window.Intl === undefined) {
```

Comment:
Hmm, should this go somewhere into src/lib/ ?  AFAIR, `Intl` is used in src/lib/context.